pkg/storage/trb_request_attendee.go

```diff
@@ -0,0 +1,106 @@
+package storage
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+
+	"github.com/google/uuid"
+
+	"go.uber.org/zap"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/apperrors"
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+// CreateTRBRequestAttendee creates a new TRB request attendee record in the database
+func (s *Store) CreateTRBRequestAttendee(ctx context.Context, attendee *models.TRBRequestAttendee) (*models.TRBRequestAttendee, error) {
+	attendee.ID = uuid.New()
+	const createTRBRequestAttendeeSQL = `
+		INSERT INTO trb_request_attendees (
+			id,
+			eua_user_id,
+			trb_request_id,
+			role,
+			component,
+			created_by,
+			modified_by
+		)
+		VALUES (
+			:id,
+			:eua_user_id,
+			:trb_request_id,
+			:role,
+			:component,
+			:created_by,
+			:modified_by
+		)`
```

Comment:
We also now have `pkg/storage/SQL` to hold SQL, rather than needing to have it in-line in our store methods - could you move these SQL statements into that folder and embed them here?